ietf-optical-impairment-topology.tree

```diff
@@ -28,15 +29,21 @@ module: ietf-optical-impairment-topology
                 |  +--ro amplifier
                 |     +--ro type-variety    string
                 |     +--ro operational
-                |        +--ro actual-gain                             decimal64
-                |        +--ro tilt-target                             decimal64
-                |        +--ro out-voa                                 decimal64
-                |        +--ro in-voa                                  decimal64
+                |        +--ro actual-gain
+                |        |       decimal64
+                |        +--ro tilt-target
+                |        |       decimal64
+                |        +--ro out-voa
+                |        |       decimal64
+                |        +--ro in-voa
+                |        |       decimal64
                 |        +--ro (power-param)?
                 |           +--:(channel-power)
-                |           |  +--ro nominal-channel-power?            decimal64
+                |           |  +--ro nominal-channel-power?
+                |           |          decimal64
                 |           +--:(power-spectral-density)
-                |              +--ro nominal-power-spectral-density?   decimal64
+                |              +--ro nominal-power-spectral-density?
+                |                      decimal64
```

Comment:
what is the frequency slot used for measurement?

Reply:
not related to the pull-request

Reply:
To be addressed in open issue #37